### PR TITLE
[bug] docker-compose.yaml starts postgres but does not use it

### PR DESCRIPTION
**Pull request by desmax74**
# Description of the PR
docker-compose.yaml starts postgres but does not use it.
see : https://github.com/guacsec/guac/issues/1386

# PR Checklist

- [x] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [ ] All new changes are covered by tests
- [ ] If GraphQL schema is changed, `make generate` has been run
- [ ] If `collectsub` protobuf has been changed, `make proto` has been run
- [x] All CI checks are passing (tests and formatting)
- [ ] All dependent PRs have already been merged

### CODE REVIEW

File: container_files/guac/guac.yaml [hunk 1]
```diff
@@ -34,3 +34,4 @@ interval: 5m
 db-driver: postgres
 db-address: postgres://guac:guac@postgres:5432/guac?sslmode=disable
 db-migrate: true
+db-debug: false
```
Comment (pxp928) [inline container_files/guac/guac.yaml:37]: what is this change for?

**Review by pxp928 — APPROVED** [review]
Thanks! Per the issue, we need a better way to bring up the databases based on the backends selected.

File: container_files/guac/guac.yaml (hunk 1 above)
Comment (mihaimaruseac) [reply container_files/guac/guac.yaml:37]: +1 to this

**Review by mihaimaruseac — APPROVED** [review]

**Review by lumjjb — APPROVED** [review]

File: container_files/guac/guac.yaml (hunk 1 above)
Comment (desmax74) [reply container_files/guac/guac.yaml:37]: @pxp928 is a missing param in the db configuration https://github.com/guacsec/guac/blob/main/pkg/cli/store.go#L64

Comment (pxp928) [reply container_files/guac/guac.yaml:37]: hmmm looks like they are not being used. I think we should remove it for the time being if not needed. Same with `db-migrate`?

Comment (desmax74) [reply container_files/guac/guac.yaml:37]: @pxp928 `db-migrate` param is used https://github.com/guacsec/guac/blob/main/pkg/assembler/backends/ent/migrate/schema.go 
I'll remove only the `db-debug`

**Pull request merged**